src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -538,6 +538,24 @@ describe('ReactDOMComponent', function() {
       expect(console.error.argsForCall[0][0]).toContain('clipPath');
       expect(console.error.argsForCall[0][0]).toContain('clip-path');
     });
+    
+    it('should only warn once about deprecated SVG attributes', function() {
+      spyOn(console, 'error');
+      var container = document.createElement('div');
+      ReactDOM.render(
+        <svg clipPath="0 0 100 100">
+          <rect strokeWidth={1} />
+          <rect strokeWidth={10} />
```

Comment:
This is correct, but a little subtle.  Usually the way we solve this is to have a second invocation of `ReactDOM.render`.  It's not immediately obvious that the second `rect` tag is adding anything; I could easily see someone going through the code and "cleaning it up".  But again, this is correct and this nit may not be worth fixing, just something to keep in mind in the future I suppose.